gulpfile.js

```diff
@@ -31,6 +24,7 @@ var gulp = require('gulp'),
     getFileInfo = require('./src/gulp-tasks/fileInfo').getFileInfo,
     buildXSLT = require('./src/gulp-tasks/buildXSLT')(gulp, plugins),
     exportBlocks = require('./src/gulp-tasks/exportBlocks')(gulp, plugins),
+    commitStatusOnExport = require('./src/gulp-tasks/commitStatusOnExport')(gulp, plugins, config),
```

Comment:
I wonder if there's a more appropriate name for this, to indicate that it's actually generating a file rather than just getting the status? Perhaps something like `exportStatus`?